doc/mgr/dashboard.rst

```diff
@@ -69,29 +72,36 @@ aspects of your Ceph cluster:
 * **Cluster logs**: Display the latest updates to the cluster's event and
   audit log files. Log entries can be filtered by priority, date or keyword.
 * **Hosts**: Display a list of all hosts associated to the cluster, which
-  services are running and which version of Ceph is installed.
+  disk are attached, which services are running and which version of Ceph is
+  installed.
 * **Performance counters**: Display detailed service-specific statistics for
   each running service.
 * **Monitors**: List all MONs, their quorum status, open sessions.
-* **Monitoring**: Enables creation, re-creation, editing and expiration of
-  Prometheus' Silences, lists the alerting configuration of Prometheus and
-  currently firing alerts. Also shows notifications for firing alerts. Needs
-  configuration.
+* **Monitoring**: Enable creation, re-creation, editing and expiration of
+  Prometheus' Silences, list the alerting configuration of Prometheus and all
+  configured and firing alerts. Show notifications for firing alerts.
 * **Configuration Editor**: Display all available configuration options,
   their description, type and default values and edit the current values.
 * **Pools**: List all Ceph pools and their details (e.g. applications,
-  placement groups, replication size, EC profile, CRUSH ruleset, etc.)
+  autoscaling, placement groups, replication size, EC profile, CRUSH rulesets,
+  quotas etc.)
 * **OSDs**: List all OSDs, their status and usage statistics as well as
   detailed information like attributes (OSD map), metadata, performance
   counters and usage histograms for read/write operations. Mark OSDs
   up/down/out, purge and reweight OSDs, perform scrub operations, modify
   various scrub-related configuration options, select different profiles to
-  adjust the level of backfilling activity.
+  adjust the level of backfilling activity. List all disk associated with an
+  OSD. Set and change the device class of an OSD, display and sort OSDs by
+  device class. Deploy new OSDs on new disks/hosts.
+* **Device management**: List all hosts known by the orchestrator. List all
+  disks and their properties attached to a node. Display disk health information
+  (health prediction and SMART data). Blink enclosure LEDs.
 * **iSCSI**: List all hosts that run the TCMU runner service, display all
   images and their performance characteristics (read/write ops, traffic).
-  Create, modify and delete iSCSI targets (via ``ceph-iscsi``). See
-  :ref:`dashboard-iscsi-management` for instructions on how to configure this
-  feature.
+  Create, modify and delete iSCSI targets (via ``ceph-iscsi``). Display the
+  iSCSI gateway status on the landing page and info about active initiators.
+  See :ref:`dashboard-iscsi-management` for instructions on how to configure
+  this feature.
 * **RBD**: List all RBD images and their properties (size, objects, features).
```

Comment:
For RBD, we also have:
* Support RBD namespace
* Allow deleting images with snapshots (https://github.com/ceph/ceph/pull/33067)

Reply:
Added, thank you!